app/packages/app/src/useEvents/useSetFieldVisibility.ts

```diff
@@ -0,0 +1,14 @@
+import { useSessionSetter } from "@fiftyone/state";
```

Comment:
This looks like duplication. This can be removed in favor of `useSetFieldVisibilityStage.ts`